htmd/projections/metriccoordinate.py

```diff
@@ -71,26 +86,26 @@ def project(self, mol):
         mol = mol.copy()
         if self.pbc:
             mol.wrap(self._centersel)
-        mol.align(trajalnsel, refmol=self._refmol, refsel=self._refalnsel)
+
+        if trajalnsel is not None and self._refmol is not None:
+            mol.align(trajalnsel, refmol=self._refmol, refsel=self._refalnsel)
 
         coords = np.concatenate((mol.coords[atomsel, 0, :], mol.coords[atomsel, 1, :], mol.coords[atomsel, 2, :]))
         return np.transpose(coords)  # I need to permute the dimensions here to put frames on the first
 
     def _getSelections(self, mol):
```

Comment:
this is not getting, it is more calculating

Reply:
I don't understand this comment, sorry :( what is getting/calculating?